### PR TITLE
Modularization/displace core

### PR DESCRIPTION
**Pull request by jtlap**
just putting core at par to ease the following steps

### CODE REVIEW

File: include/eve/function/sincosd.hpp [hunk 1]
```diff
@@ -14,6 +14,6 @@ namespace eve
   EVE_MAKE_CALLABLE(sincosd_, sincosd);
 }
 
-#include <eve/module/real/core/function/scalar/sincosd.hpp>
-//#include <eve/module/real/core/function/regular/simd/sincosd.hpp>
+#include <eve/module/core/function/scalar/sincosd.hpp>
+//#include <eve/module/core/regular/impl/simd/sincosd.hpp>
```
Comment (jfalcou) [inline include/eve/function/sincosd.hpp:18]: why the comment ?

File: include/eve/function/slice_low.hpp [hunk 2]
```diff
@@ -14,6 +14,6 @@ namespace eve
   EVE_MAKE_CALLABLE(slice_low_, slice_low);
```
Comment (jfalcou) [inline include/eve/function/slice_low.hpp:14]: this whole file can be deleted.

File: include/eve/function/slice_high.hpp [hunk 3]
```diff
@@ -14,6 +14,6 @@ namespace eve
   EVE_MAKE_CALLABLE(slice_high_, slice_high);
```
Comment (jfalcou) [inline include/eve/function/slice_high.hpp:14]: this whole file can be deleted.

File: include/eve/function/sincosd.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/function/sincosd.hpp:18]: the file as to be suppressed  sindcosd is the good one
I'll do it

**Pull request merged**